metrix-simulator/cmake/tnr.cmake

```diff
@@ -10,8 +10,7 @@
 # 
 
 function(check_file file expected_file)
-    configure_file(${file} ${file} NEWLINE_STYLE LF) # required for windows ctest
-    execute_process( COMMAND ${CMAKE_COMMAND} -E compare_files ${file} ${expected_file}
+    execute_process( COMMAND ${CMAKE_COMMAND} -E compare_files --ignore-eol ${file} ${expected_file}
```

Comment:
changed this way because previous code was assuming code was checked out using LF end of files, but this is not the default git behavior on Windows.

Note that `--ignore-eol` requires cmake 3.14+